usecases/MinimumWorkingExample/emodul/validation_test/graph.ttl

```diff
@@ -0,0 +1,37 @@
+@prefix ex: <http://example.org/ns#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <http://schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix dash: <http://datashapes.org/dash#> .
+@prefix schema: <http://schema.org/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix ns1: <http://www.w3.org/2002/07/owl#> .
+@prefix ns2: <http://www.ontologyrepository.com/CommonCoreOntologies/> .
+@prefix ns3: <http://purl.obolibrary.org/obo/> .
+@prefix ns4: <http://www.geneontology.org/formats/oboInOwl#> .
+@prefix ns5: <http://www.daml.org/2003/01/periodictable/PeriodicTable#> .
+@prefix ns6: <http://purl.org/dc/elements/1.1/> .
+@prefix ns7: <https://www.materials.fraunhofer.de/ontologies/BWMD_ontology/mid#> .
+@prefix ns8: <https://www.materials.fraunhofer.de/ontologies/graph_designer#> .
+@prefix ns9: <http://purl.org/dc/terms/> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix mseo: <https://purl.matolab.org/mseo/mid/> .
+@prefix cco: <http://www.ontologyrepository.com/CommonCoreOntologies/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix con: <http://w3id.org/concrete/> .
+@prefix tt: <http://w3id.org/concrete/youngs/> .
+
+tt:Diam a cco:Diameter ;
+    obo:RO_0010001 tt:DiamVal .
+
+tt:DiamVal a cco:InformationBearingEntity ;
+    cco:has_decimal_value 12.5 ;
+    cco:uses_measurement_unit [ a cco:MeasurementUnitOfLength ] .
+
+tt:Spec a mseo:Specimen ;
+    cco:is_input_of [ a cco:YoungsModulusTest ] ;
+    cco:designated_by [ a cco:NonNameIdentifier ] ;
+    obo:BFO_0000051 [ obo:RO_0000086 [] ] .
```

Comment:
I guess the shape.ttl is the standard validation file for all Emodul (better Young's modulus) tests, thus it should go (and then to be extended) into LebeDigital/lebeDigital/shacl/youngs_modulus_shape.ttl. The graph.ttl that you are using is for testing purposes, thus it should be stored in LebeDigital/test/shacl/youngs_modulus.ttl

Reply:
I've renamed the file and created another CPTO_shape.ttl which resides in the same folder as the validation code, testing specific properties about the CPT Ontology.